### DISCUSSION TITLE
[Support]: semantic search link broken

### DISCUSSION THREAD

**Discussion by moogoos** [blakeblackshear/frigate#15123] [General Support] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

Under "search settings" The "Read the documentation" link is broken. Either change it or setup a redirect on:

https://docs.frigate.video/configuration/semantic_search

This is on the /settings page

### Steps to reproduce

See description

### Version

15-2 beta

### Frigate config file

```yaml
NA
```

### docker-compose file or Docker CLI command

```yaml
NA
```

### Relevant Frigate log output

```shell
NA
```

### Relevant go2rtc log output

```shell
NA
```

### Operating system

Other Linux

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

NA

### Screenshots of the Frigate UI's System metrics pages

NA

**Comment by hawkeye217** [comment]
This is not a bug. The link will point to the correct place after 0.15 comes out of beta.

**Comment by hawkeye217** [comment, marked as answer]
This is the temporary link to the beta documentation: https://deploy-preview-13787--frigate-docs.netlify.app/configuration/semantic_search/
